curvefs/src/tools/create/curvefs_create_topology_tool.cpp

```diff
@@ -415,6 +436,118 @@ int CurvefsBuildTopologyTool::GetServersInZone(
     return 0;
 }
 
+int CurvefsBuildTopologyTool::ClearPool() {
+    TopologyService_Stub stub(&channel_);
+    for (auto it : poolToDel) {
+        DeletePoolRequest request;
+        DeletePoolResponse response;
+        request.set_poolid(it);
+
+        brpc::Controller cntl;
+        cntl.set_timeout_ms(FLAGS_rpcTimeoutMs);
+        cntl.set_log_id(1);
```

Comment:
if you don't want to use `log_id` to trace RPC calls, there's no need to set it. otherwise, you have to set a different ID for each RPC call.

Reply:
> if you don't want to use `log_id` to trace RPC calls, there's no need to set it. otherwise, you have to set a different ID for each RPC call.

fix